### ISSUE TITLE
Building fail

### ISSUE THREAD

**Issue by aleith** [StrangeLoopGames/EcoIssues#64]
alpha 3.0, Mac OS X 10.10.5 
Right clicking second co-ordinate creates an impression of the correct dimension (sometimes) but Eco misinterprets the dimensions — by a lot! 

here are some examples. I managed to get my first attempt at this correct but due to lack of timber I (accidentally) abandoned the building permit. can't make another one, most frustrating.

https://www.dropbox.com/s/65welq041j18otm/Screenshot%202016-01-30%2016.45.09.png?dl=0
https://www.dropbox.com/s/g9dkgxh6pe2htj9/Screenshot%202016-01-30%2016.44.33.png?dl=0
https://www.dropbox.com/s/r02u76k2rbvam18/Screenshot%202016-01-30%2016.43.56.png?dl=0
https://www.dropbox.com/s/hlgm62j70pv14zo/Screenshot%202016-01-30%2016.41.59.png?dl=0

altogether as a zip file:
https://www.dropbox.com/s/556ckqb9qhwpybm/screenshots-of-building-fail.zip?dl=0  

Also the in-game feedback after a fail is non-useful.

**Comment by klimSLG** [comment]
Sorry for having hard time on building.
Building area is indicate is interior area which border of grid is not part of it.
If area is too thin is not enough or too vague to understand what's wrong, what you want to suggest?
will "Interior must be X(Area), border is not included" this be clear?

**Comment by aleith** [comment]
Well it took me a while to realise that one unit of length in buildings terms is two units as indicated by the cube-form targeting HUD and the voxel units the game is rendered in (in foreground at least). So i don't know why there's that duality — i have no Minecraft priors. When I got my perimeter claimed I then set about building the foundation as directed (3x3) to one unit height and it never got me onto the next instruction, very frustrating spent a few hours mucking around just learning how to move around and get back up the hill when i fall off etc. I'll reserve my critiques until I actually can accomplish things in the game, but I came here for an exercise in ecosystem balance and sustainable development and I'm feeling like there's to much schlepping around before I get to the main course so far. I realise the details are what make any game grippy but, yeah, frustrating at this stage, maybe the finished game needs a good walk through to start players off if they haven't got Minecraft experience.
